contracts/ICS20Transfer.sol

```diff
@@ -100,12 +104,17 @@ contract ICS20Transfer is
     }
 
     /// @inheritdoc IICS20Transfer
-    function sendTransfer(IICS20TransferMsgs.SendTransferMsg calldata msg_) external override returns (uint32) {
+    function sendTransfer(IICS20TransferMsgs.SendTransferMsg calldata msg_)
+        external
+        override
+        whenNotPaused
```

Comment:
I think the "when" part is implied. I think it should be notPaused.

Reply:
This is inherited from openzeppelin, the name isn't defined by us